Classes/Message/MessageFactory.php

```diff
@@ -86,6 +87,10 @@ public function createMessage($type, Application $application)
 
         $message->setApplication($application);
 
+        // Use this signal to modify messages before they are passed to the form (for default input etc.)
+        $signalSlotDispatcher = $this->objectManager->get(Dispatcher::class);
+        list($message) = $signalSlotDispatcher->dispatch(__CLASS__, 'afterMessageCreated', [$message, $this]);
```

Comment:
Is passing the whole factory to the slot intentional?

Reply:
Removed. The factory is stateless anyway and can be instanciated if needed. Thanks :)